acceptance-tests/src/main/kotlin/org/ostelco/at/common/StripePayment.kt

```diff
@@ -66,6 +69,29 @@ object StripePayment {
         return source.id
     }
 
+    fun createInsuffientFundsPaymentSourceId(): String {
+
+        // https://stripe.com/docs/api/java#create_source
+        Stripe.apiKey = System.getenv("STRIPE_API_KEY")
+
+        val sourceMap = mapOf(
+                "type" to "card",
+                "card" to mapOf(
+                        "number" to "4000000000000341",
+                        "exp_month" to 12,
+                        "exp_year" to nextYear(),
+                        "cvc" to "314"),
+                "owner" to mapOf(
+                        "address" to mapOf(
+                                "city" to "Oslo",
+                                "country" to "Norway"
+                        ),
+                        "email" to "me@somewhere.com")
+        )
+        val source = Source.create(sourceMap)
```

Comment:
I would have preferred calling this paymentSource, just to make it blindingly obvious to casual readers (such as myself, in the role of reviewer) what this value is.

Reply:
Good point!